### PR TITLE
Add Code blocks in configure-service-account.md

### PR DESCRIPTION
**Pull request by ydFu**
* Add Code blocks in the Markdown spec to make it easy to read.

* Uniform case and adjustment description for smoother writing.

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md [hunk 1]
```diff
@@ -199,9 +202,9 @@ The content of `token` is elided here.
     ```
 
 - Verify it has been created.
-   ```shell
-   kubectl get secrets myregistrykey
-   ```
+    ```shell
+    kubectl get secrets myregistrykey
+    ```
```
Comment (tengqm) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md:207]: This additional indentation is antipattern. Please change it back.

Comment (ydFu) [reply content/en/docs/tasks/configure-pod-container/configure-service-account.md]: Thank,I will change it back.

**Comment by ydFu** [comment]
@tengqm 
Thanks for your advice,I adjusted the content.

**Comment by ydFu** [comment]
/assign @savitharaghunathan
Pls review this PR,Thank you.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-25969--kubernetes-io-master-staging.netlify.app/docs/tasks/configure-pod-container/configure-service-account/

**Comment by kbhawkey** [comment]
Thanks @ydFu .
/approve

**Pull request merged**